app/src/main/java/org/mozilla/fenix/tabstray/viewholders/SyncedTabsPageViewHolder.kt

```diff
@@ -5,32 +5,40 @@
 package org.mozilla.fenix.tabstray.viewholders
 
 import android.view.View
-import androidx.recyclerview.widget.GridLayoutManager
 import androidx.recyclerview.widget.RecyclerView
+import mozilla.components.feature.syncedtabs.view.SyncedTabsView
+import mozilla.components.lib.state.ext.observeAsComposableState
 import org.mozilla.fenix.R
-import org.mozilla.fenix.databinding.ComponentSyncTabsTrayLayoutBinding
+import org.mozilla.fenix.databinding.ComposeSyncTabsTrayLayoutBinding
 import org.mozilla.fenix.tabstray.TabsTrayStore
+import org.mozilla.fenix.tabstray.syncedtabs.SyncedTabsList
+import org.mozilla.fenix.theme.FirefoxTheme
 
 class SyncedTabsPageViewHolder(
     containerView: View,
-    private val tabsTrayStore: TabsTrayStore
+    private val tabsTrayStore: TabsTrayStore,
+    private val listener: SyncedTabsView.Listener
```

Comment:
Wondering if this should just be `TabClickDelegate` over its interface?

Reply:
I don't recall why i did it this way (possibly a holdover from the original code). I think it makes sense to have it be `TabClickDelegate`.